src/i18n.js

```diff
@@ -126,7 +152,33 @@ const messages = {
     'navbar.about': '정보t',
     'navbar.docs': '문서',
     'navbar.faq': 'FAQ',
-    'footer.copyright': '© 2023 EOS 네트워크 재단 판권 소유.'
+    'footer.copyright': '© 2023 EOS 네트워크 재단 판권 소유.',
+
+    'app.walletconnect.failedswitch.title': '지갑에 네트워크 구성이 누락되었습니다',
+    'app.walletconnect.failedswitch.content': `<p>귀하가 사용하려는 지갑이 아직 EOS EVM을 지원하지 않은 것으로 확인되었습니다. 다음 네트워크 정보를 이용하여 네트워크를 추가하거나 <a href="https://docs.eosnetwork.com/evm/quick-start/introduction" >문서 페이지</a> 를 방문하여 자세한 내용을 확인해 보시기 바랍니다.</p>
```

Comment:
Please use the following link for KO https://docs.eosnetwork.com/ko/evm/quick-start/setup-metamask/

Reply:
done